tests/e2e/specs/setup-mc/step-2-product-listings.test.js

```diff
@@ -418,9 +418,6 @@ test.describe( 'Configure product listings', () => {
 		} );
 
 		test( 'should see the heading of next step and send two requests after clicking "Continue"', async () => {
-			const requestPromises =
```

Comment:
I believe the `/gla/mc/contact-information` request is still being fetched, so instead of asserting the policy check response, could we assert the contact information instead? Otherwise, we're just asserting the heading without checking any of the requests. Also, the title could be updated to align with these changes.

Reply:
Sounds good @jorgemd24 . I've now updated the PR for additional tests.